src/sql/workbench/api/browser/mainThreadQueryEditor.ts

```diff
@@ -145,4 +148,9 @@ export class MainThreadQueryEditor extends Disposable implements MainThreadQuery
 	public $setQueryExecutionOptions(fileUri: string, options: azdata.QueryExecutionOptions): Thenable<void> {
 		return this._queryManagementService.setQueryExecutionOptions(fileUri, options);
 	}
+
+	public async $tryCreateQueryDocument(options?: { content?: string }, providerId?: string): Promise<URI> {
```

Comment:
why is there a `try` in the name?

Reply:
Good point, I took this from the create text document stuff but there doesn't seem to be a reason for that name here